src/electron/platform/android/rule-information.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { RuleResultsData } from './scan-results';
+
+export type HowToFixDelegate = (ruleResultsData: RuleResultsData) => string;
+
+export class RuleInformation {
+    private RuleId: string;
+    private RuleDescription: string;
+    private HowToFixDelegate: HowToFixDelegate;
```

Comment:
We might be able to initialize these directly in the constructor like the link [here](https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties). We have some examples in our codebase if you look for "private readonly" inside a constructor

Reply:
Refactored as suggested